bucket/bizhawk.json

```diff
@@ -0,0 +1,91 @@
+{
+    "version": "2.4.2",
+    "description": "Multi system emulator",
+    "homepage": "http://tasvideos.org/BizHawk.html",
+    "license": "MIT",
+    "##": [
+        "The requirements are listed in https://github.com/TASVideos/BizHawk-Prereqs/blob/master/README",
+        "The list of firmware-specific directories to persist comes from the base directories listed in BizHawk's /src/BizHawk.Client.Common/config/PathEntryCollection.cs"
+    ],
+    "suggest": {
+        "Microsoft Visual C++ Redistributables 2015": "extras/vcredist2015"
+    },
+    "notes": ".NET Framework 4.8 is required.",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/TASVideos/BizHawk/releases/download/2.4.2/BizHawk-2.4.2.zip",
+            "hash": "0e642e54add4a1533525cb4cad5fcfa3d1e1234225321db28a1fbbaba98fd247"
+        }
+    },
+    "bin": [
+        "EmuHawk.exe",
+        [
+            "EmuHawk.exe",
+            "BizHawk"
+        ],
+        "DiscoHawk.exe"
+    ],
+    "shortcuts": [
+        [
+            "EmuHawk.exe",
+            "BizHawk"
+        ],
+        [
+            "DiscoHawk.exe",
+            "DiscoHawk"
+        ]
+    ],
+    "persist": [
+        "config.ini",
+        "shaders",
+        "Firmware",
+        "Movies",
+        "Tools",
+        "Lua",
+        "ExternalTools",
+        "Intellivision",
+        "NES",
+        "SNES",
+        "GBA",
+        "SMS",
+        "Game Gear",
+        "SG-1000",
+        "Genesis",
+        "PC Engine",
+        "Gameboy",
+        "Dual Gameboy",
+        "TI83",
+        "Atari 2600",
+        "Atari 7800",
+        "C64",
+        "ZXSpectrum",
+        "AmstradCPC",
+        "PSX",
+        "Coleco",
+        "N64",
+        "Saturn",
+        "WonderSwan",
+        "Lynx",
+        "Apple II",
+        "Libretro",
+        "VB",
+        "NGP",
+        "PCFX",
+        "ZXSpectrum",
+        "GB3x",
+        "GB4x",
+        "VEC",
+        "O2",
+        "MSX"
```

Comment:
```suggestion
        "AmstradCPC",
        "Apple II",
        "Atari 2600",
        "Atari 7800",
        "C64",
        "Coleco",
        "Dual Gameboy",
        "ExternalTools",
        "Firmware",
        "Game Gear",
        "Gameboy",
        "GB3x",
        "GB4x",
        "GBA",
        "Genesis",
        "Intellivision",
        "Libretro",
        "Lua",
        "Lynx",
        "Movies",
        "MSX",
        "N64",
        "NES",
        "NGP",
        "O2",
        "PC Engine",
        "PCFX",
        "PSX",
        "Saturn",
        "SG-1000",
        "shaders",
        "SMS",
        "SNES",
        "TI83",
        "Tools",
        "VB",
        "VEC",
        "WonderSwan",
        "ZXSpectrum",
        "ZXSpectrum"
```

Reply:
I added this, but I left shaders/Firmware/Movies/Tools/Lua/ExternalTools at the top next to `config.ini` because like `config.ini` they are not part of the architecture-specific folders